frontend/src/components/ActivityAssessmentDetails/ActivityAssesmentDetailsStyles.js

```diff
@@ -0,0 +1,139 @@
+import styled from 'styled-components';
+import { Col, Row, Button } from 'react-bootstrap';
+
+export const ContentCol = styled(Col)`
+    padding-top: 15px;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
Nie sprawdzałem ile wynoszą marginy po y. Jeżeli są równe 0 to zapisz:
```css
margin: 0 auto;
```
Jeżeli są domyślne > 0 to pomiń ten komentarz